doc/config.md

```diff
@@ -58,7 +58,7 @@ When `reverse = true` is set for data-table, there are two possibles cases for [
 Depends on it to have a proper retention and aggregation you must set `rollup-use-reverted = true` for the first case and `rollup-use-reverted = false` for the second.
 
 ## Carbonlink `[carbonlink]`
-The configuration to get metrics from carbon-cache. See details in [graphite-web](https://graphite.readthedocs.io/en/latest/carbon-daemons.html#carbon-relay-py) documentation.
+The configuration to get the fresh metrics from `carbon-cache`. See details in [graphite-web](https://graphite.readthedocs.io/en/latest/carbon-daemons.html#carbon-relay-py) documentation.
```

Comment:
I think this is still not correct.

`carbon-cache` is a daemon there that writes whisper files and can communicate all non-written datapoints via carbonlink.
This option enabled carbonlink listener, that allows to trick graphite-web to think this is just a normal carbon-cache that just happens to have tons of points in memory and nothing on disk.

Reply:
As discussed over telegram - apparantly I misunderstood what this feature is doing and it's not very clear when it's useful. So that's fine to leave it like that and wait for a real-world user to describe when this is helpful and add that to the docs later.